OSPSuite.Presentation.Importer/Core/IDataSource.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace OSPSuite.Presentation.Importer.Core
+{
+   /// <summary>
+   /// Collection of DataSets
+   /// </summary>
+   public interface IDataSource
+   {
+      IDictionary<string, IDataSet> DataSets { get; set; }
+   }
+
+   public class DataSource : IDataSource
+   {
+      public IDictionary<string, IDataSet> DataSets { get; set; } = new Dictionary<string, IDataSet>();
```

Comment:
A lot of members have setters as well. Why?

Reply:
Specific setter removed, I ll check also for setters of other members of they can be removed.